bindings/pyroot/pythonizations/test/uhi_indexing.py

```diff
@@ -0,0 +1,253 @@
+"""
+Tests to verify that TH1 histograms conform to the UHI setting and accessing interfaces.
+"""
```

Comment:
Thank you, this is a very good start for the testing suite! I believe we may need tests that also check the statistics of the slices, e.g. the number of entries, the mean, the stddev etc. If some things are still to be understood in that area, we can leave it for a follow-up PR.